lib/common/xml.c

```diff
@@ -1315,6 +1315,23 @@ write_xml_file(const xmlNode *xml, const char *filename, gboolean compress)
     return (int) nbytes;
 }
 
+/*!
+ * \internal
+ * \brief Check whether a string's first two bytes represent a Unicode charcter
+ *
+ * \param[in] text  String to check
+ *
+ * \return \c true if the first two bytes of \p text represent a Unicode
+ *         character, or \c false otherwise
+ */
+static inline bool
+is_unicode(const char *text)
+{
+    // @TODO Is this a valid and complete test for multi-byte Unicode chars?
```

Comment:
Mostly: https://stackoverflow.com/questions/5012803/test-if-char-string-contains-multibyte-characters

I think the only problem is assuming there will be an even number of bytes. It can be 2, 3, or 4.